library/Fission/Web/App/Destroy.hs

```diff
@@ -0,0 +1,66 @@
+module Fission.Web.App.Destroy
+  ( API
+  , ByURLAPI
+  , ByIdAPI
+  , server
+  , destroyById
+  , destroyByURL
+  ) where
+
+import           Database.Esqueleto
+import           Servant
+
+import           Fission.Prelude
+import           Fission.Models
+import           Fission.URL.Types
+
+import qualified Fission.App.Destroyer.Class as App
+import           Fission.Web.Error.Class
+
+type API = ByURLAPI :<|> ByIdAPI
+
+type ByURLAPI
+  =  "associated"
+  :> Capture "url" URL
```

Comment:
how does servant tell the difference between a URL & an AppId if they're both strings?

Reply:
It'll check for `"associated"` first

```
/app/associated/foo.com
/app/12345
```

Reply:
ah got it got it :+1: